xapian-core/cluster/lc.cc

```diff
@@ -39,7 +39,8 @@ LC::LC(unsigned int k_)
 {
     LOGCALL_CTOR(API, "LC", k_);
     if (k_ == 0)
-	throw InvalidArgumentError("Number of required clusters should be greater than zero");
+	throw InvalidArgumentError("Number of required clusters should be \
+				    greater than zero");
```

Comment:
You don't want to do this, because it inserts a lot of spaces into the string. Instead, you can use multiple string constants; they'll be concatenated together at compile time. So something like:

```c++
    throw InvalidArgumentError("First string constant starts here "
                               "second string constant just continues");
```

Note the trailing space in the first string constant. See for instance `xapian-core/tests/harness/backendmanager.cc:230` or so.

Reply:
Could you please elaborate a bit more on "inserts a lot of spaces into the string"?

Reply:
Think about what your string constant actually contains. Before the word "greater" there will be a lot of spaces.

Reply:
Oh, I get what you mean. The backslash escapes the newline and the spaces before 'greater' get included in the string, which is not the intended effect. Thanks for the tip.